fe/fe-core/src/main/java/com/starrocks/rpc/RpcCallBackImpl.java

```diff
@@ -0,0 +1,22 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.rpc;
+
+import com.baidu.brpc.client.RpcCallback;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+
+public class RpcCallBackImpl<T> implements RpcCallback<T> {
+    private static final Logger LOG = LogManager.getLogger(RpcCallBackImpl.class);
+
+    @Override
+    public void success(T response) {
```

Comment:
need to release the ResponseBinaryAttachment here, according to the [document](https://github.com/baidu/brpc-java/blob/master/docs/cn/client.md#1%E5%AE%9A%E4%B9%89%E5%BC%82%E6%AD%A5callback) of brpc-java.
Correct me if I'm wrong.

Reply:
I confirmed that, it is needed.